src/services/TileService.ts

```diff
@@ -66,13 +63,18 @@ export class TileService {
 
         this.textureArray = new Array<WebGLTexture>(numTextures);
         this.resetCoordinateQueue();
-
         this.cachedTiles = new LRUCache<number, RasterTile>(Int32Array, null, lruCapacityGPU);
-        this.persistentTiles = new Map<number, RasterTile>();
-        this.pendingRequests = new Map<number, boolean>();
 
         // L2 cache: compressed tiles on system memory
         this.lruCapacitySystem = lruCapacitySystem;
+    };
+
+    constructor(backendService: BackendService, lruCapacityGPU: number = 512, lruCapacitySystem: number = 4096) {
+        this.backendService = backendService;
+        this.channelMap = new Map<number, { channel: number, stokes: number }>();
+        this.setCache(lruCapacityGPU, lruCapacitySystem);
```

Comment:
The problem with this is that it will create a large cache in the constructor, and then update it when preferences arrive. We should either just initialise the cache with the lowest possible values, or work out how to avoid re-initialising it altogether.

Reply:
@duidae I've updated your branch with a minor change that solves this, please have a look and see  if you agree with it

Reply:
got it, thanks!